### PR TITLE
remove dead links

### PR DESCRIPTION
**Pull request by Swati4star**
remove dead links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -232,7 +224,6 @@ Powershell, Redis and lots others. Also has a good collection of IDEs for progra
 * [Codechef IDE](https://www.codechef.com/ide) - Simple single file code compiler supporting major languages.
 * [Cloud9](https://c9.io/) `[Account]`  - Very powerful online dev environment, just like your local PC.
 * [JS Bin](https://jsbin.com) - Collaborative web development and debugging tool.
-* [Codepen.io](https://codepen.io/) - Playground for front end web development.
```
Comment (aviaryan) [inline README.md]: It looks like you used an automated tool to do this. However, I don't understand how codepen.io got flagged here. Can you please check this?

Comment (Swati4star) [reply README.md]: It was showing in the list of issues here https://travis-ci.org/aviaryan/awesome-no-login-web-apps/builds/596221771

Let me revert this change & try build again

Comment (Swati4star) [reply README.md]: added to whitelist

**Comment by Swati4star** [comment]
@aviaryan Is this fine now?

**Comment by Swati4star** [comment]
Done @aviaryan

**Pull request merged**